### ISSUE TITLE
Controller doesn't inhibit idle

### ISSUE THREAD

**Issue by ghost** [nowrep/wljoywake#3]
I am trying to use this to inhibit idle, but it seems that whenever I run the command, it registers the controller fine, but idle is not inhibited. Is there any way to more verbose output? There are no errors or anything else when ran normally, so I can't really figure out what this issue is.

**Comment by nowrep** [comment]
What's your compositor?

**Comment by ghost** [comment]
riverwm, I am pretty sure it used to work about a year ago, but haven't really played games with a gamepad since then, so I might be mistaken.

**Comment by nowrep** [comment]
Can you try on sway?

**Comment by ghost** [comment]
Yes, it works one sway. However, I talked with the dev of river and he said, that sway has implemented the protocol incorrectly and that [this PR](https://github.com/swaywm/sway/pull/8175) will cause wljoywake to stop working on sway as well.
There is a bit more information about it on [this issue](https://codeberg.org/river/river/issues/1079).

**Comment by nowrep** [comment]
Yeah that makes sense. Not much to do here other than use completely different mechanism of idle inhibit (maybe systemd).

**Comment by ghost** [comment]
Yea, seems logind or elogind, in case of a different init, might be the only real solution

**Comment by ghost** [comment]
this tool has now ceased to work on sway with the 1.11 release. pipewire idle inhibit has fixed it with: https://github.com/rafaelrc7/wayland-pipewire-idle-inhibit/pull/18
perhaps wljoywake could do the same?
